components/pedigree/resources/src/main/resources/pedigree/view/personHoverbox.js

```diff
@@ -125,6 +139,10 @@ define([
             $super();
 
             this.generateMenuBtn();
+            if (this.getNode().getLifeStatus() == "alive" || this.getNode().getLifeStatus() == "deceased") {
+                this.generateAliveWell();
+            }
+            this.getBoxOnHover().attr({'height': PedigreeEditorParameters.attributes.personHoverBoxRadius * 2 + this.getLabelsOffset()});
```

Comment:
well, `getLabelesOffset()` is supposed to say somethign about labels, not about hoverboxes. If you treat it as described, you shoudl rename it to `hoverBoxSize()`